src/index.mako.html

```diff
@@ -721,80 +722,64 @@
 
     <script>
       (function() {
+        function getParam(name){
+          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
+        }
+
+
         // Make sure ajax is using cached requests
         $.ajaxSetup({
           cache: true
         });
-
+ 
+        var adminUrlRegexp = new RegExp('${admin_url_regexp}');
         var module = angular.module('geoadmin');
         var cacheAdd = '${version}' != '' ? '/' + '${version}' : '';
         var pathname = location.pathname.replace(/(index|mobile|embed)\.html$/g, '');
-
-        function getParam(name){
-          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
+        var staging = 'prod';
+        var testres = /^mf-geoadmin3\.([a-z]+)\.bgdi.ch$/.exec(location.hostname);
```

Comment:
That's ok when served behind apache but in S3 what would the expected behaviour be?
We'll probably have only a unique address so what should be the default env?